felix/bpf-apache/sockops.c

```diff
@@ -15,13 +15,13 @@
 #include <linux/bpf.h>
 #include "sockops.h"
 
-struct bpf_map_def __attribute__((section("maps"))) calico_sk_endpoints = {
-	.type           = BPF_MAP_TYPE_LPM_TRIE,
-	.key_size       = sizeof(union ip4_bpf_lpm_trie_key),
-	.value_size     = sizeof(__u32),
-	.max_entries    = 65535,
-	.map_flags      = BPF_F_NO_PREALLOC,
-};
+struct {
+    __uint(type, BPF_MAP_TYPE_LPM_TRIE);
+    __uint(max_entries, 65535);
+    __type(key, union ip4_bpf_lpm_trie_key);
+    __type(value, __u32);
+    __uint(map_flags, BPF_F_NO_PREALLOC);
+} calico_sk_endpoints __attribute__((section(".maps")));
```

Comment:
@debasishbsws this is correct, but please replace it with `SEC()` macro to align with other files.

Reply:
Like the one here: https://github.com/projectcalico/calico/blob/5741d7df6dfe2453c41be46f4d990dd7b56b1d4c/felix/bpf-apache/filter.h#L28

Reply:
@debasishbsws I went ahead, and fixed this. Thanks for reporting this.